src/lib/modules/ping/flood_ping.go

```diff
@@ -154,9 +155,8 @@ func (ps *FloodPing) ping(urlRaw string, packets int, tsdbNameHash string, wg *s
 		wg.Done()
 		return
 	}
-
-	result := *scrap.CLIFLoodPingScrap(resp)
-	newBlock := *tsdb.GetNewBlock("flood-ping", getNormalizedBlockStringFlood(result))
+	result := scrap.CLIFLoodPingScrap(resp)
```

Comment:
I got the reason why it was used as value `*`. 

The returned value is an address, so it should rather be as `*(scrap.CLIFLoodPingScrap(resp))` if we want it was done previously. Else, we can go with the current change as well. I don't think there will be any change in performance.

Reply:
We go in as per the current